components/org.wso2.carbon.identity.verification.onfido.connector/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
We shouldn't change the license year of existing files. Check other places as well

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/12/commits/95c30c53a7755ddb13b54ce67aa26e1abfb47327